src/AWS.Deploy.Recipes/DeploymentBundleDefinitions/Container.deploymentbundle

```diff
@@ -11,6 +11,26 @@
             "AdvancedSetting": true,
             "Updatable": true
         },
+        {
+            "Id": "DockerfilePath",
+            "Name": "Dockerfile Path",
+            "Description": "Specifies the absolute path to the Dockerfile as an alternative to the default location.",
```

Comment:
Why does it have to be an absolute path. Eventually we want to persist build settings like we do with the other recipe settings and when we do that we will want to persist the file path relative to the project path. Otherwise we deployment would get messed up as projects are deployed from different machines.

Reply:
I brought this up at a internal stand-up this past week and nobody raised concerns.

I agree supporting absolute or relative would be nice. The biggest obstacle is that a `IOptionSettingItemValidator` doesn't have access to the project path per https://github.com/aws/aws-dotnet-deploy/blob/main/docs/OptionSettingsItems-input-validation.md#dependencies. I'll see if I can extend that to be aware of the project path.